index.js

```diff
@@ -11,7 +11,8 @@ async function runAction() {
   const token = core.getInput('humanitec-token', {required: true});
   const orgId = core.getInput('organization', {required: true});
   const imageName = core.getInput('image-name') || process.env.GITHUB_REPOSITORY.replace(/.*\//, '');
-  const dockerfile = core.getInput('dockerfile') || '.';
+  const dockerfile = core.getInput('dockerfile') || '';
```

Comment:
I think it is very common to specify a directory rather than a Dockerfile.
The change as it stands is not backwards compatible.

Reply:
The --file  argument specifies a file, not a directory
A directory specified is a context directory (but it’s not necessary that Dockerfile should be in this path).
In my opinion `dockerfile` name is confusing if it’s used to define a context path.

https://docs.docker.com/engine/reference/commandline/build/#text-files